### ISSUE TITLE
Fix CI on OPERATOR_NOT_ALLOWED_IN_QUERY

### ISSUE THREAD

**Issue by dhmlau** [strongloop/loopback-connector-mongodb#503]
## Description
Currently there is one test case failed in CI because of the error below:

```
mongodb connector
does not execute a nested $where:
Uncaught AssertionError: expected Error {
code: 'OPERATOR_NOT_ALLOWED_IN_QUERY',
statusCode: 400,
details: Object {
operators: Array [ '$where' ],
where: Object {
content: Object {
$where: 'function() {return this.content.contains("content")}'
}
}
},
message: 'Operators "$where" are not allowed in query'
} to not exist
at Post.find (test/mongodb.test.js:830:24)
at /home/travis/build/strongloop/loopback-connector-mongodb/node_modules/loopback-datasource-juggler/lib/dao.js:1525:7
at process._tickCallback (internal/process/next_tick.js:61:11)

```

_Extracted from https://github.com/strongloop/loopback-connector-mongodb/pull/497_

There is a setting in mongodb, there is a setting : allow extended operators 
In juggler,  there are security settings that may have contributed to this failure (introduced ~ 3 months ago)

Related to this change : https://github.com/strongloop/loopback-connector-mongodb/pull/483/files#diff-f44d7a6206dcaa5ddd05b31118b5c2beR786 

## Acceptance Criteria
- [ ] Fix CI and make it all green again
  - [ ] `master` branch
  - [ ] `3.x` branch

**Comment by dhmlau** [comment]
Also see https://github.com/strongloop/loopback-datasource-juggler/pull/1705#issuecomment-480254509

**Comment by dhmlau** [comment]
See PR #505

**Comment by b-admike** [comment]
Fixed by #505, created #509 for follow up removal of downstream builds.
